libraries/rhsm_errata_level.rb

```diff
@@ -0,0 +1,36 @@
+#
+# Author:: Chef Partner Engineering (<partnereng@chef.io>)
+# Copyright:: Copyright (c) 2015 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module RhsmCookbook
+  class RhsmErrataLevel < ChefCompat::Resource
+    include RhsmCookbook::RhsmHelpers
+
+    resource_name :rhsm_errata_level
+
+    property :errata_level, kind_of: String, name_property: true
+
+    action :install do
+      validate_errata_level!(errata_level)
```

Comment:
Totally OT, but are errata levels case sensitive?  If so, lame, but I'd love to see us support lowercase goodness.  If not, lets make that helper downcase all the things.